src/components/ha-button.ts

```diff
@@ -0,0 +1,20 @@
+import { ButtonBase } from "@material/mwc-button/mwc-button-base";
+import { styles } from "@material/mwc-button/styles.css";
+import { css } from "lit";
+import { customElement } from "lit/decorators";
+
+@customElement("ha-button")
+export class HaButton extends ButtonBase {
+  static override styles = [
+    styles,
+    css`
+      // TODO : add custom style
+    `,
```

Comment:
We probably should not do this, but just try to implement `md-text-button`, `md-filled-button`, `md-tonal-button`, `md-outlined-button`, etc

Reply:
https://github.com/material-components/material-web/tree/master/button

Reply:
https://www.npmjs.com/package/@material/web

Reply:
But we have to keep the same properties as mwc-button to avoid issues or we change all the properties everywhere ? 😅

Reply:
Well... yeah... that's something we have to decide on... We either let `ha-button` handle the conversion, or have to do a lot more work...